src/components/PaymentRequestButtonElement.test.js

```diff
@@ -72,8 +77,11 @@ describe('PaymentRequestButtonElement', () => {
     );
 
     expect(elementMock.on.mock.calls[0][0]).toBe('ready');
-    expect(onReadyMock).toHaveBeenCalled();
+    expect(onReadyMock.mock.calls).toHaveLength(1);
     expect(elementRefMock).toHaveBeenCalledWith(elementMock);
```

Comment:
Nitpicky: can we also include the `toHaveBeenCalledWith` for the `onReadyMock` to prove we passed it in?

Reply:
Whoops, didn't even see that we could do the same thing we're doing for elementRefMock! Good catch, thanks 😄